packages/session-replay-browser/src/helpers.ts

```diff
@@ -25,3 +25,8 @@ export const isSessionInSample = function (sessionId: number, sampleRate: number
   const mod = absHashMultiply % 100;
   return mod / 100 < sampleRate;
 };
+
+export const getCurrentUrl = () => {
+  // eslint-disable-next-line no-restricted-globals
+  return typeof window !== 'undefined' ? window.location.href : '';
```

Comment:
Can you use `getGlobalScope` here instead? You won't have to disable the lint rule then too and its a bit more standard in the codebase

Reply:
Yeah I can. Did not realize there was a function for this already.